### ISSUE TITLE
Create a Deck card out of a chat message

### ISSUE THREAD

**Issue by jakobroehrl** [nextcloud/talk-android#1048]
This is really nice implemented in the Browser version:
![image](https://user-images.githubusercontent.com/24530680/109812069-e3125700-7c2b-11eb-813d-746d37998238.png)
![image](https://user-images.githubusercontent.com/24530680/109812109-ee658280-7c2b-11eb-8050-26b6911eb635.png)

Could we implement this in the App, too? 
Maybe calling the Deck App (if installed) with the message content for creating a new card there? @stefan-niedermann

**Comment by stefan-niedermann** [comment]
Yeah, sounds nice. We can theoretically go the same way [like with the integration with the Files app notifications](https://github.com/nextcloud/android/blob/master/src/main/java/com/nextcloud/client/integrations/deck/DeckApiImpl.java): Providing an intent which you can hook on and fallback to opening the web browser with the given URL.

In fact we already provide two kind of relevant intents: "Create new card" as well as "Attach to existing card". Let me know if you need more information.

Of course the Talk app should check whether Deck is installed and active on this instance (or only check whether the Android Deck app is installed :thinking: ?)
